Basic example compilation error

Compilation of the following example test_pythran.py
"""
#pythran export mytest(float[])
def mytest(n):
    res = (n+2*pow(n**2,2))/n
    return res
"""
with command : 
>pythran test_pythran.py

fails with the following error:

"""
/tmp/tmpwq5KHw.cpp:57:110:   required from here
/usr/local/lib/python2.7/dist-packages/pythran/pythonic/include/types/tuple.hpp:870:37: error: no type named 'type' in 'struct std::tuple_element<0, std::tuple<> >'
           std::tuple<Tys...>> ::type;
                                     ^
WARNING: Compilation error, trying hard to find its origin...
WARNING: Nop, I'm going to flood you with C++ errors!
CRITICAL: Cover me Jack. Jack? Jaaaaack!!!!
E: error: Command "x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-IbFBHb/python2.7-2.7.16=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DENABLE_PYTHON_MODULE -D__PYTHRAN__=2 -DPYTHRAN_BLAS_BLAS -I/usr/local/lib/python2.7/dist-packages/pythran -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/local/include -I/usr/include -I/usr/include/python2.7 -c /tmp/tmpwq5KHw.cpp -o /tmp/tmpb_5iui/tmp/tmpwq5KHw.o -std=c++11 -fno-math-errno -w -fvisibility=hidden -fno-wrapv" failed with exit status 1
"""
Pythran version : 0.9.3
Python version : 2.7.16
Operating System: Debian GNU/Linux 10 (buster)
Kernel: Linux 4.19.0-5-amd64
Architecture: x86-64

--- a/docs/CLI.rst
+++ b/docs/CLI.rst
@@ -29,7 +29,7 @@ The generated native ``.so`` module can then be used with the Python interpreter
 Pythran version can be dumped through ``--version``::
 
   $> pythran --version 2>&1
-  0.9.3
+  0.9.3post0
 
 The module-level ``__pythran__`` variable indicates that the module loaded has been pythranized::
 

--- a/pythran/pythonic/include/types/numpy_broadcast.hpp
+++ b/pythran/pythonic/include/types/numpy_broadcast.hpp
@@ -269,7 +269,7 @@ namespace types
     using value_type = dtype;
     using const_iterator = const_broadcast_iterator<dtype>;
     using iterator = const_iterator;
-    static constexpr size_t value = 0;
+    static constexpr size_t value = 1;
 
     broadcast_base<dtype, is_vectorizable> _base;
     operator dtype() const

--- a/pythran/tests/test_numpy_broadcasting.py
+++ b/pythran/tests/test_numpy_broadcasting.py
@@ -195,3 +195,13 @@ class TestBroadcasting(TestEnv):
                       np.arange(100.),
                       np.arange(100.),
                       broadcast_sum_axis=[NDArray[float, :], NDArray[float, :]])
+
+    def test_broadcasting_nth29(self):
+        code = '''
+        def broadcast_nth29(n):
+            res = (n+2*pow(n**2,2))/n
+            return res'''
+        self.run_test(code,
+                      np.arange(100.),
+                      broadcast_nth29=[NDArray[float, :]])
+

--- a/pythran/version.py
+++ b/pythran/version.py
@@ -1,3 +1,3 @@
-__version__ = '0.9.3'
+__version__ = '0.9.3post0'
 __url__ = 'https://github.com/serge-sans-paille/pythran'
 __descr__ = 'Ahead of Time compiler for numeric kernels'

